__init__.py

```diff
@@ -90,6 +90,10 @@ def initialize(self):
         self.gui.register_handler("mycroft.device.confirm.tts", self.select_tts)
         self.nato_dict = self.translate_namedvalues('codes')
 
+        # trigger initial pairing
+        if not is_paired():
+            self.bus.emit(Message("mycroft.not.paired"))
```

Comment:
this is just a round trip to https://github.com/OpenVoiceOS/skill-ovos-setup/blob/dev/__init__.py#L83 shouldn't it self.not_paired({})

Reply:
i should not be blocking, emiting a message makes this event based

otherwise skill loading would block in initialize method instead of loading next skill